### PR TITLE
Add get_level_values for Index and MultiIndex 

### PR DESCRIPTION
**Pull request by itholic**
Take over #972 

```python
mi = pd.MultiIndex.from_arrays((list('abc'), list('def')))
mi.names = ['level_1', 'level_2']
kdf = ks.DataFrame({'a': [1, 2, 3]}, index=mi)

# get values by level number
kdf.index.get_level_values(0)

# get values by level number
kdf.index.get_level_values('level_1')
```

### CODE REVIEW

File: databricks/koalas/indexes.py [hunk 1]
```diff
@@ -2498,6 +2517,82 @@ def __getattr__(self, item: str) -> Any:
...
+        lev = self.levels[level]
+        name = self.names[level]
+
+        return ks.Index(lev, name=name)
+
+    def get_level_values(self, level):
+        """
+        Return vector of label values for requested level,
+        equal to the length of the index.
+        Parameters
```
Comment (HyukjinKwon) [inline databricks/koalas/indexes.py:2570]: Let's add spaces for each section

Comment (itholic) [reply databricks/koalas/indexes.py:2555]: Thanks :)

File: databricks/koalas/indexes.py [hunk 2]
```diff
@@ -2498,6 +2518,78 @@ def __getattr__(self, item: str) -> Any:
                 return partial(property_or_func, self)
         raise AttributeError("'MultiIndex' object has no attribute '{}'".format(item))
 
+    def _get_level_number(self, level):
+        """
+        Return the level number if a valid level is given.
+        **this is an internal method**
```
Comment (HyukjinKwon) [inline databricks/koalas/indexes.py:2524]: No need to document this because leading `_` already means that.

File: databricks/koalas/indexes.py [hunk 3]
```diff
@@ -2498,6 +2518,78 @@ def __getattr__(self, item: str) -> Any:
...
+        """
+        count = self.names.count(level)
+        if (count > 1) and not isinstance(level, int):
+            raise ValueError("The name %s occurs multiple times, use a level number" % level)
+        try:
+            level = self.names.index(level)
```
Comment (HyukjinKwon) [inline databricks/koalas/indexes.py:2530]: I think you can use if-else instead of relying on try-catch.

**Comment by HyukjinKwon** [comment]
Looks good otherwise.

File: databricks/koalas/indexes.py (hunk 2 above)
Comment (itholic) [reply databricks/koalas/indexes.py:2524]: Agree, I'm gonna remove this

File: databricks/koalas/indexes.py (hunk 3 above)
Comment (itholic) [reply databricks/koalas/indexes.py:2529]: Thanks @HyukjinKwon , Fixed!

**Pull request merged**